eng/pipelines/source-build-sdk-diff-tests.yml

```diff
@@ -4,7 +4,20 @@ schedules:
   branches:
     include:
     - main
-    - release/*
+    - release/*.0.1xx*
+    - internal/release/*.0.1xx*
+    exclude:
+    - release/3*

```

Comment:
Why are the 3 and 5 EOL versions necessary here?  This pipeline yml doesn't exist for those versions.

Reply:
Ok, that makes sense. I wasn't sure what the behavior would be. In that case I can get rid of 6 and 7 branches as well since the YAML file doesn't exist in those either.